shaka-lab-gateway-client/README.md

```diff
@@ -7,54 +7,11 @@ Lab Gateway to log in.
 [`shaka-lab-gateway`](https://github.com/shaka-project/shaka-lab/tree/main/shaka-lab-gateway#readme)
 package.  Devices with the client package will connect to that gateway.
 
+This package is available for:
```

Comment:
Do we still need the note about the gateway package in this file, if we also reproduce it in each platform-specific doc?

Reply:
You can't use the gateway-client package without the gateway package running on Linux somewhere on the network.  I think if you're looking at the detailed doc for a specific platform, the note helps.  (Though I'm open to debate and changes if you disagree.)